src/box/lua/console.lua

```diff
@@ -611,7 +613,11 @@ local function local_read(self)
             if not lang or lang == 'lua' then
                 -- stop once a complete Lua statement is entered
                 if local_check_lua(buf) then
-                    break
+                    if not line:endswith(multiline_delim) then
+                        break
+                    end
+                    buf = buf:sub(1, -2)
```

Comment:
Nit: Since we have `multiline_delim` as a variable, it would be consistent to make `buf = buf:sub(1, -1 - #multiline_delim)` here (and below in `client_read()`).

Reply:
Thx, added that.